ResourceManager/Strings.cs

```diff
@@ -53,6 +62,9 @@ public static void Reinitialize()
         public static string InstallGitInstructions => _instance.Value._installGitInstructions.Text;
         public static string GitExecutableNotFound => _instance.Value._gitExecutableNotFoundText.Text;
 
+        public static string TelemetryPersmissionCaption => _instance.Value._telemetryPersmissionCaption.Text;
+        public static string TelemetryPersmissionMessage => _instance.Value._telemetryPersmissionMessage.Text;
```

Comment:
Replace all `elemetryPersmission` with `elemetryPermission`.

Reply:
Thank you!